Bundle/BlogBundle/Resources/views/Blog/Tabs/_articles.html.twig

```diff
@@ -25,6 +25,7 @@
                 <a class="vic-panelBlog-title" href="{{vic_business_link(article)}}">
                     {{ article.name }}
                 </a>
+                <em>{{ article.author }}</em>
```

Comment:
The default toString displays the username: https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Model/User.php#L678